codes/typescript/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this config file necessary?

Reply:
I was thinking to add some proper test cases into the `typescript`, but it's gonna be fine to remove it.

Reply:
What is the relationship between the test case and this config file?